osquery/tables/system/windows/appcompat_shims.cpp

```diff
@@ -31,71 +31,74 @@ QueryData genShims(QueryContext& context) {
   QueryData shimResults;
   std::map<std::string, sdb> sdbs;
 
-  queryKey("HKEY_LOCAL_MACHINE",
-           "SOFTWARE\\Microsoft\\Windows "
+  queryKey("HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows "
            "NT\\CurrentVersion\\AppCompatFlags\\InstalledSDB",
            sdbResults);
   for (const auto& rKey : sdbResults) {
     QueryData regResults;
     sdb sdb;
-    std::string subkey = rKey.at("subkey");
-    auto start = subkey.find("{");
-    if (start == std::string::npos) {
-      continue;
-    }
-    std::string sdbId = subkey.substr(start, subkey.length());
-    // make sure it's a sane uninstall key
-    queryKey("HKEY_LOCAL_MACHINE", subkey, regResults);
-    for (const auto& aKey : regResults) {
-      if (aKey.at("name") == "DatabaseDescription") {
-        sdb.description = aKey.at("data");
-      }
-      if (aKey.at("name") == "DatabaseInstallTimeStamp") {
-        // take this crazy windows timestamp to a unix timestamp
-        sdb.installTimestamp = std::stoull(aKey.at("data"));
-        sdb.installTimestamp = (sdb.installTimestamp / 10000000) - 11644473600;
-      }
-      if (aKey.at("name") == "DatabasePath") {
-        sdb.path = aKey.at("data");
+    if (rKey.at("type") == "subkey") {
```

Comment:
Ah, thanks for the note. Looks like git saw this as a change as the indent level increased?